Light Rework

PatRat2 opened the pull request:
Added recipes for all unimplemented lighting components, nerfed the arc lamp to 1 per craft

I hope the stoichiometry is good, and I know some elements like YAG:Ce and semiconductor chemicals will be added by other PRs at some point

Also waiting for PVD machine to be added from the semiconductor rework so I put a metallization recipe into the CVD machine temporarily

PatRat2 committed:
Light Rework

Added recipes for all unimplemented lighting components, nerfed the arc lamp to 1 per craft

I hope the stoichiometry is good, and I know some elements like YAG:Ce and semiconductor chemicals will be added by other PRs at some point

Rick8484 commented on groovy/postInit/components/EtchablesChain.groovy:
```diff
@@ -564,3 +567,50 @@ ASSEMBLER.recipeBuilder()
+    .EUt(VA[HV])
+    .buildAndRegister()
+
+CVD.recipeBuilder()
```
Duplicated Recipe ?

PatRat2 replied:
ayeup

PatRat2 committed:
Remove duplicate recipe

planetme commented on groovy/material/SecondDegreeMaterials.groovy:
```diff
@@ -1052,5 +1052,49 @@ class SecondDegreeMaterials {
+                .color(0xd6d1a9)
+                .build()
+
+        TrimethylGalliumSolution = new Material.Builder(13174, SuSyUtility.susyId('trimethyl_gallium_solution'))
```
I think trimethylgallium and trimethylindium are one word

trainvoi commented on groovy/postInit/chemistry/ChemistryOverhaul.groovy:
```diff
@@ -3870,4 +3870,42 @@ CRYSTALLIZER.recipeBuilder()
+
+BR.recipeBuilder()
+    .fluidInputs(fluid('trimethylaluminium') * 1000)
+    .fluidInputs(fluid('gallium_trichloride') * 1000)
```
GaCl3 and InCl3 are solids at room temperature, you need to vaporize it first?

PatRat2 replied:
GaCl3 and InCl3 are soluble in toluene for this

PatRat2 committed:
Added 2 more lamps

low pressure and high pressure mercury vapor lamps, high pressure for lighting and low pressure for UV

PatRat2 committed:
Added 2 more lamps

low pressure and high pressure mercury vapor lamps, high pressure for lighting and low pressure for UV

PatRat2 committed:
Revert "Added 2 more lamps"

This reverts commit e4b91996a3db98abb48d94f342992f63a48e2192.

PatRat2 committed:
Also cerium dioxide

for YAG:Ce for LEDs

Rick8484 commented on groovy/postInit/mod/GregTech.groovy:
```diff
@@ -1962,3 +1953,176 @@ ALLOY_SMELTER.recipeBuilder()
+    .duration(400)
+    .buildAndRegister()
+
+ROASTER.recipeBuilder()
```
You can put these type of recipes in either their respective chains (ceriumchain) or ChemistryOverhaul.groovy

Rick8484 commented:
also tbh you could just put all of these recipes in a new groovy file 
makes groovy slaves life easier by 7.56%

PatRat2 committed:
Moved recipes around to respective files

PatRat2 committed:
I mixed files up, the graphics are back

Rick8484 commented on groovy/postInit/chemistry/inorganic_chemistry/elements/f_block/lanthanides/CeriumChain.groovy:
```diff
@@ -84,3 +84,13 @@ ROASTER.recipeBuilder()
+ROASTER.recipeBuilder()
+    .fluidInputs(fluid('oxygen') * 2000)
+    .inputs(metaitem('dustCerium'))
+    .outputs(metaitem('dustCeriumIvOxide'))
```
Just realized
The amount of atom = amount of dust
so 3 dust it is

PatRat2 committed:
Fixed cerium + OreDict

PatRat2 committed:
Also an argonless recipe for UV lamp

even though really it sucks but its better than filling the tube with nitrogen or something

PatRat2 committed:
Oops messed up stoichiometry

PatRat2 committed:
Merge branch 'master-ceu' into master-ceu

PatRat2 committed:
Recipe to turn old Carbon Arc lamps into Incandescent Ones

PatRat2 committed:
Merge branch 'master-ceu' into master-ceu

PatRat2 committed:
Merge branch 'master-ceu' into master-ceu

PatRat2 committed:
Merge remote-tracking branch 'origin/master-ceu' into master-ceu

PatRat2 committed:
Fixed the raw LED wafer recipe

thanks bru :)

PatRat2 committed:
Removed the running message

trainvoi committed:
Merge branch 'master-ceu' into master-ceu

trainvoi commented on groovy/material/SecondDegreeMaterials.groovy:
```diff
@@ -1048,5 +1048,49 @@ class SecondDegreeMaterials {
+                .color(0xd6d1a9)
+                .build()
+
+        TrimethylGalliumSolution = new Material.Builder(13174, SuSyUtility.susyId('trimethyl_gallium_solution'))
```
move the trimethyl chemicals to Organic Materials

PatRat2 committed:
Added compressor recipes for seed oil

PatRat2 committed:
Moved trimethyl chemicals to OrganicChemistryMaterials

PatRat2 committed:
Readded a missing recipe

PatRat2 committed:
Readded a missing recipe

PatRat2 committed:
Merge pull request #1 from PatRat2/patch-1

Patch 1

PatRat2 committed:
Decapitalizing json stuff

PatRat2 committed:
Rename etched_LED_wafer.json to etched_led_wafer.json

PatRat2 committed:
Rename LED_chip.json to led_chip.json

PatRat2 committed:
Rename metallized_LED_wafer.json to metallized_led_wafer.json

PatRat2 committed:
Rename patterned_LED_wafer.json to patterned_led_wafer.json

PatRat2 committed:
Rename raw_LED_wafer.json to raw_led_wafer.json

PatRat2 committed:
Rename etched_LED_wafer.png to etched_led_wafer.png

PatRat2 committed:
Rename LED_chip.png to led_chip.png

PatRat2 committed:
Rename metallized_LED_wafer.png to metallized_led_wafer.png

PatRat2 committed:
Rename patterned_LED_wafer.png to patterned_led_wafer.png

PatRat2 committed:
Rename raw_LED_wafer.png to raw_led_wafer.png

bruberu committed:
Merge branch 'master-ceu' into master-ceu

bruberu committed:
fix: capitalization
